services/realtimeclock.md

```diff
@@ -0,0 +1,33 @@
+# Real time clock
+
+    
+    identifier: 0x1a8b1a28
+    extends: _sensor
+
+## Registers
+
+    ro now @ reading {
```

Comment:
do we also want a reading as seconds since epoch? I guess we then need to synchronize between the two readings. not sure about that one

Reply:
Most of these clocks use Y2K as the base time; we could add an offset to meet epoch - but since it's not going to be milliseconds, i taught it was fine. Maybe ``now`` is not a good name.

Reply:
I was thinking of another integer register; I'm fine with y2k. it would be mostly useful to measure passage of time - eg your JD device goes to sleep, wakes up, and you want to know how long you were out - computing date->seconds is quite complicated, so it would be nice if the service gave a monotonic second counter

Reply:
we could ditch this blow-out reading and use epoch always, let the clients do the math.

Reply:
we could, but the math is somewhat involved, with leap years and the like; and the hardware typically uses y/m/d/h/m/s, just like you do here.